ajax/libs/leaflet-locatecontrol/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "leaflet-locatecontrol",
+  "filename": "L.Control.Locate.min.js",
+  "version": "0.52.0",
+  "description": "A useful control to geolocate the user with many options. Official Leaflet and MapBox plugin.",
+  "homepage": "https://github.com/domoritz/leaflet-locatecontrol",
```

Comment:
Would you please remove homepage field since it's the same as the repo, and we will automatically use repository for it, thanks.